GETTING_STARTED.md

```diff
@@ -0,0 +1,125 @@
+You will notice in the 3d view there is a new square which has been added that is black. This would be incorrect in real life, as the source is likely not going to be hitting the sample straight away. To remedy this we should add a transformation to the newly added source component.
+
+#### Setting translation on the source
+
+To the right of the Add component button in the main window toolbar, three additional controls can be used to add transformations to a component. Here we have the option to translate, rotate and link to another component's transformations. 
+
+In this case, we will add a translation, so we can move the source to it's correct position. To do this, first select the source component in the list by clicking it, then click the "add translation" button.
+
+You will see upon adding a new translation the black cube gets shifted to the right immediately. This is because of the default vector specified in the translation. By default the field is disabled, but to edit these fields simply click on it. When finished editing, clicking elsewhere in the list will save the changes. 
+
+![](resources/images/NeXusConstructor_008.png) 
+
+As the source distance will be static and its position will likely not change throughout the experiment, we can set this to a scalar value. A realistic value would have a vector of (0,0,1) and the distance would be `-20m`. Distance units can be changed with the units field next to the value. For this value a float data type should be used, which is the default anyway. 
+
+![](resources/images/NeXusConstructor_009.png)
+
+You will notice after inputting values that the black cube moves behind the sample cube. This is because it is now -20 metres behind it on the z-axis. 
+
+### Adding a detector 
+
+*Detectors* are a key component of any beamline, as they detect the speed and intensity of neutrons/muons/photons after (or before) the beam has passed through the sample. We can use a grid of detectors to formulate an image of the scattered beam.  
+
+Detectors can be added in the same way as other components, but can contain *pixel data* in addition to their shape. When a detector or detector module component type is selected and shape information is inputted, the pixel data options will appear.
+This is important because some detectors are made up of a grid of pixels, and this needs to be portrayed in the constructor to show the position of each. In our case, we will set the shape type to cylinder and the outcome should look like this:
+
+
+#### Setting pixel data
+
+In this example we will say our detector is comprised of a 16x16 grid of "pixels" or individual detectors. 
+
+To replicate this in the NeXus Constructor, first create a component with `NXdetector` as it's component type. After this, select `Mesh` as the geometry type. This will show the pixel data edits like so:
```

Comment:
```suggestion
Select `Cylinder` as the "Shape type".
```
It is too confusing to select cylinder options and then switch to mesh. Also you've said you are selecting mesh and then show a screenshot of cylinder options. Can you change this to only talk about cylinders for the detector and instead add a new section which adds a monitor with a mesh shape?